development/cli/getting_started.rst

```diff
@@ -64,14 +64,14 @@ Common options that can be used with any of phpBB's CLI commands.
    :header: "Option", "Usage"
    :delim: |
 
-   --help (-h) | Display a help message
-   --quiet (-q) | Do not output any message
-   --verbose (-v,-vv,-vvv) | Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
-   --version (-V) | Display this application version
-   --ansi | Force ANSI (colors) output
-   --no-ansi | Disable ANSI (colors) output
-   --no-interaction (-n) | Do not ask any interactive question
-   --safe-mode | Run in Safe Mode (without extensions)
+   ``--help (-h)`` | Display a help message
+   ``--quiet (-q)`` | Do not output any message
+   ``--verbose (-v,-vv,-vvv)`` | Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
+   ``--version (-V)`` | Display this application version
+   ``--ansi`` | Force ANSI (colors) output
+   ``--no-ansi`` | Disable ANSI (colors) output
+   ``--no-interaction (-n)`` | Do not ask any interactive question
+   ``--safe-mode`` | Run in Safe Mode (without extensions)
```

Comment:
Instead of parenthesis, use commas. For example:
```
``--verbose, -v, -vv, -vvv``
```

Reply:
Both done.